.github/workflows/ci.yaml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - README.md
+  pull_request:
+    paths-ignore:
+      - README.md
+
+jobs:
+    php:
+        name: PHP ${{ matrix.php }} Symfony ${{ matrix.symfony }}
+        runs-on: ubuntu-18.04
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [7.4, 8.0, 8.1]
+                symfony: [4.4.*, 5.2.*, 5.3.*, 5.4.*]
```

Comment:
5.2 and 5.3 are not supported anymore. 
And you targetet 5.4 component in the composer.json, does it not lead to any conflict ?